packages/story-editor/src/components/panels/design/sizePosition/radius.js

```diff
@@ -207,7 +207,10 @@ function RadiusControls({ selectedElements, pushUpdateForObject }) {
       </StackableGroup>
 
       <LockContainer>
-        <Tooltip title={__('Toggle consistent corner radius', 'web-stories')}>
+        <Tooltip
+          title={__('Toggle consistent corner radius', 'web-stories')}
+          key={borderRadius.locked.toString()}
```

Comment:
good pr description about this - super smart. Should we leave a comment here explaining what it does?

```js
// Remounts the component when `borderRadius.locked` changes
```